qiskit/transpiler/passes/utils/check_map.py

```diff
@@ -69,11 +69,10 @@ def run(self, dag):
                 self.property_set["is_swap_mapped"] = False
                 return
         for cf_instr in dag.op_nodes(op=ControlFlowOp):
-            layout = Layout(dict(enumerate(cf_instr.qargs)))
+            order = [dag.qubits.index(bit) for bit in cf_instr.qargs]
```

Comment:
Rather than using `list.index`, you might want to use the `qubit_indices` map that's defined a few lines above, to avoid a ~~quadratic dependency in building this list.

Reply:
Fixed in 9462293f5.